src/services/zapper.ts

```diff
@@ -275,7 +275,7 @@ export class ZapperService extends Service {
       sellToken = ZeroAddress;
     }
 
-    const url = `https://api.zapper.fi/v1/zap-in/vault/${zapProtocol}/transaction`;
+    const url = `https://api.zapper.fi/v2/zap-in/vault/${zapProtocol}/transaction`;
```

Comment:
The result has changed, refactored with https://github.com/yearn/yearn-sdk/pull/284/commits/acc8eaa40bd605a023f370a4f3860b0a06a5d4ea

Reply:
Actually it appears the API still returns that but it's undocumented

![telegram-cloud-photo-size-5-6287230568680763069-y](https://user-images.githubusercontent.com/78794805/165416043-060e2ff7-f2e3-4ecf-b7f2-4434c7966aa0.jpg)

Reply:
Reverted back the change because we are still getting the `buyTokenAddress`, but probably should be careful as it's undocumented and can be removed without notice.

Reply:
Agree with you we should rather not use `buyTokenAddress`. For a zapIn tx, it should always be same as the vault address instead of that one. `targetTokenAddress` should be the vault address.

Reply:
I was thinking doing that refactor in another PR @xgambitox, maybe the one that adds Zod. What are your thoughts?